### PR TITLE
Feature/legacy redirect

### PR DESCRIPTION
**Pull request by skoszuta**

### CODE REVIEW

File: src/components/MetadataProvider/MetadataProvider.tsx [hunk 1]
```diff
@@ -2,11 +2,13 @@ import { type ReactNode, createContext, useEffect, useState } from "react";
 
 const METADATA_HOST = "";
 const METADATA_JSON = `${METADATA_HOST}/metadata.json`;
+const LEGACY_READER_HOST = "https://old-graypaper.fluffylabs.dev"; // todo: replace with actual url
```
Comment (tomusdrw) [inline src/components/MetadataProvider/MetadataProvider.tsx:5]:
```suggestion
const LEGACY_READER_HOST = "https://old-graypaper.fluffylabs.dev";
```

File: src/components/Version/Version.tsx [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
-  const handleChange = useCallback(
-    (version: string) => {
+  const handleChange = useCallback<ChangeEventHandler<HTMLSelectElement>>(
+    (e) => {
+      const version = e.target.value;
+
+      if (metadata.versions[version].legacy) {
+        window.open(urlGetters.legacyReaderVersion(version), "_blank");
+        return;
+      }
```
Comment (tomusdrw) [inline src/components/Version/Version.tsx:23]: I think it's better not to display the legacy versions at all. We just need to be able to redirect old links.

File: src/components/MetadataProvider/MetadataProvider.tsx (hunk 1 above)
Comment (skoszuta) [reply src/components/MetadataProvider/MetadataProvider.tsx]: outdated, fixed

File: src/components/Version/Version.tsx (hunk 2 above)
Comment (skoszuta) [reply src/components/Version/Version.tsx:23]: oh alright, then ill need to remove some code

Comment (skoszuta) [reply src/components/Version/Version.tsx]: done

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**